lib/acts_as_taggable_on/taggable/tagged_with_query/query_base.rb

```diff
@@ -48,7 +48,7 @@ def tags_match_type
 
     def escaped_tag(tag)
       tag = tag.downcase unless ActsAsTaggableOn.strict_case_match
-      tag.gsub(/[!%_]/) { |x| '!' + x }
+      ActsAsTaggableOn::Utils.escape_like(tag)
```

Comment:
The previous code was identical to ActsAsTaggableOn::Utils.escape_like and it seemed DRYer to funnel this all through a singular interface. But this makes the name seem odd now too.